conf/modules.config

```
@@ -27,6 +40,9 @@ process {
     }
 
     withName: 'DROP.*GENECOUNTS_MERGECOUNTS' {
+        cpus = { 30 * task.attempt }
```

from the report this process does not require that many cpus. Maybe 6?
```suggestion
        cpus = { 6 * task.attempt }
```